occupancy_grid_map.py

```diff
@@ -185,64 +185,54 @@ def init_actions(self):
                    4: np.array([[0,2], [-1,1], [0,0]]),
                    5: np.array([[-1,1], [0,1], [0,0]]),
                    6: np.array([[-1,1], [0,2], [0,0]]),
-                   7: np.array([[-1,1], [0,-1], [0,0]]), # does the negative stuff work?
-                   #7: np.array([[-1,1], [-1,0], [0,0]]), # does the negative stuff work?
-                   8: np.array([[-1,1], [0,-2], [0,0]]), # does the negative stuff work?
-                   #8: np.array([[-1,1], [-2,0], [0,0]]), # does the negative stuff work?
+                   7: np.array([[-1,1], [-1,0], [0,0]]), # does the negative stuff work?
```

Comment:
@bfaught3 to confirm do we need to change the range mentioned in the overleaf ? 
Or I am interpreting it incorrectly? 
i−1:i+1,j:j+1,k